src/tests/test_cases/test.py

```diff
@@ -67,10 +67,11 @@ def __call__(self=None, x=None, y=None, z=None):
 @expect('D103: Missing docstring in public function')
 def function():
     """ """
+    @expect('D123: Missing docstring in inaccessible public function')
     def ok_since_nested():
         pass
 
-    @expect('D103: Missing docstring in public function')
+    @expect('D123: Missing docstring in inaccessible public function')
```

Comment:
This change should be superseded by https://github.com/PyCQA/pydocstyle/pull/559